components/Calender/src/App.vue

```diff
@@ -0,0 +1,147 @@
+<template>
```

Comment:
no caps in finlename

Reply:
Vue cli default.  If we impose consistent rules for the style which I am for, they should be automated.  https://www.npmjs.com/package/file-name-linter.

Reply:
That rule may conflict though https://github.com/scbd/accounts.cbd.int/blob/master/Dockerfile

Reply:
https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended
https://angular.io/guide/styleguide
https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76
Please mark resolved or give direction to change.